eng/pipelines/templates/jobs/perf.yml

```diff
@@ -23,6 +23,10 @@ parameters:
 - name: EnvVars
   type: object
   default: {}
+- name: ToolsRepoCommitish
+  type: string
+  default: 'main'
```

Comment:
Do we really want to default to main? I most of our other references we use release tags so that changes in the main repo don't break everyone. @mikeharder are we planning to move to a release t tag model for any of the source-based assets coming from the tools repo?

Reply:
Wes and I discussed this offline.  We will probably need to move to a release tag or nuget package model eventually, but for now I think it's more efficient for the perf team as a whole to keep pulling from main.  The cost of updating each language repo (through eng/common or otherwise) when we update perf-automation would be large.  We are not yet running perf tests from hotfix branches or other non-main branches.